apps/ejabberd/src/mongoose_client_api_rooms.erl

```diff
@@ -122,6 +121,16 @@ handle_request(<<"POST">>, JSONData, Req,
             {true, RespReq, State}
     end.
 
+handle_request_by_meth(<<"POST">>, JSONData, User, Server) ->
```

Comment:
`POST` is used if you don't know the `id` of a resource you created and let the server create one. `PUT` is used when you know the `id` and what to create an item with this exact `id` which you pass in URI. The changes made in this PR doesn't reflect that, though... I'm going to change that.